src/components/AddRestaurantModal/AddRestaurantModal.jsx

```diff
@@ -1,15 +1,43 @@
+import { useState } from "react";
 import "./AddRestaurantModal.css";
 
-function AddRestaurantModal() {
+function AddRestaurantModal({ onClose, onAdd }) {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [category, setCategory] = useState("");
```

Comment:
```suggestion
  const [restaurantData, setRestaurantData] = useState({
    name: "",
    description: "",
    category: "",
  });
```
저는 여러 상태의 관심사가 같다면 하나의 상태로 묶는게 관리하기 쉽고 응집도도 높힐 수 있는 것 같다고 생각합니다~🙃

Reply:
오 좋은 방법이네요.. 확실히 가독성도 더 높아진거같아요!